tests/make_emoji_font_test.py

```diff
@@ -73,3 +73,23 @@ def test_make_emoji_font(svgs, expected_ttx, color_format, output_format):
     )
     _, ttfont = nanoemoji._generate_color_font(config, glyph_inputs)
     test_helper.assert_expected_ttx(svgs, ttfont, expected_ttx)
+
+
+@pytest.mark.parametrize(
+    "svgs", [("rect.svg", "rect2.svg"), ("one-o-clock.svg", "two-o-clock.svg")]
```

Comment:
is this what ./precommit.sh gave? (runs black)

Reply:
yes